src/client/rekor.test.ts

```diff
@@ -89,6 +102,90 @@ describe('Rekor', () => {
     });
   });
 
+  describe('#createIntoEntry', () => {
+    const artifactDigest =
+      '1c025a6e48ceb8bf10e01b367089732326eabe3541d03d348724c79040382c65';
+    const artifactSignature =
+      'MEUCIDB2SWDabztSC8RrlfRCWUf04LBN0E2CEwiDZJLacDS8AiEA3bQHMBpodxA3dvJ+JK1SALkuzju/w4oCg3S89c8CtN8=';
+    const publicKey =
+      'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K';
+
+    const proposedEntry = {
+      apiVersion: '0.0.1',
+      kind: 'hashedrekord',
```

Comment:
Should this be intoto? Looks the test case below expects it to submit a new intoto entry

```suggestion
      kind: 'intoto',
```

Reply:
Good catch -- it was actually that I'd reversed the test descriptions in the `describe` string. Fixed now.

Reply:
Did you push your changes? Looks like the below `it` still reads "submits a new intoto entry"